### ISSUE TITLE
Add OS specific commands

### ISSUE THREAD

**Issue by tyrann** [timozattol/wallpaper-generator#5]
In order to move towards a portable version of the code, I suggest to write the OS specific version of the commands to update the wallpaper, as well as the one to get the screen resolution.

**Comment by timozattol** [comment]
What do you mean? To look for the OS type and then choose the appropriate command to update wallpaper & get screen resolution?

**Comment by tyrann** [comment]
Yes,unless I'm mistaken, I don't think that `xrandr` or `grep` will work on Windows. Maybe we should look for the OS type?

**Comment by timozattol** [comment]
Yes exactly! I was just making sure I understood the issue :) 
We should probably cover Windows, Mac OS, and as many Linux desktop environment as possible. Might be hard work though
